src/init.jsx

```diff
@@ -23,7 +24,7 @@ const store = createStore(
     socket,
   }),
 
-  applyMiddleware(socketMessageHook),
+  applyMiddleware(socketMessageHook, notifications/* , logger*/),
```

Comment:
hmm maybe turn logger on if dev environment?

Reply:
i'm going to just remove it for now because mihok is not a big fan; people can put it in to their own liking when needed.